### PR TITLE
Clarify plaintext and fix punctuation

### PR DESCRIPTION
**Pull request by zekiakyol**
- It fixes more punctuations.
- It clarifies the sentences about the **nycflights13** package and its datasets.

### CODE REVIEW

File: logicals.qmd [hunk 1]
```diff
@@ -20,7 +20,7 @@ We'll finish off with `if_else()` and `case_when()`, two useful functions for ma
 ### Prerequisites
 
 Most of the functions you'll learn about in this chapter are provided by base R, so we don't need the tidyverse, but we'll still load it so we can use `mutate()`, `filter()`, and friends to work with data frames.
-We'll also continue to draw examples from the nycflights13 dataset.
+We'll also continue to draw examples from the `flights` dataset of the nycflights13 package.
```
Comment (hadley) [inline logicals.qmd:23]: Maybe `nycflights13::flights`?

File: joins.qmd [hunk 2]
```diff
@@ -17,7 +17,7 @@ This chapter will introduce you to two important types of joins:
 -   Filtering joins, which filter observations from one data frame based on whether or not they match an observation in another.
 
 We'll begin by discussing keys, the variables used to connect a pair of data frames in a join.
-We cement the theory with an examination of the keys in the nycflights13 datasets, then use that knowledge to start joining data frames together.
+We cement the theory with an examination of the keys in the datasets of the nycflights13 package, then use that knowledge to start joining data frames together.
```
Comment (hadley) [inline joins.qmd:20]:
```suggestion
We cement the theory with an examination of the keys in the datasets from the nycflights13 package, then use that knowledge to start joining data frames together.
```
?

Comment (zekiakyol) [reply joins.qmd:20]: I agree with `from`.

File: logicals.qmd (hunk 1 above)
Comment (zekiakyol) [reply logicals.qmd:23]: I agree with `nycflights13::flights`.

**Pull request merged**